src/frontend/src/lib/utils/token.utils.ts

```diff
@@ -64,16 +69,36 @@ export const mapDefaultTokenToToggleable = <T extends Token>({
 }: {
 	defaultToken: T;
 	userToken: TokenToggleable<T> | undefined;
-}): TokenToggleable<T> => ({
-	...defaultToken,
-	enabled:
+}): TokenToggleable<T> => {
```

Comment:
are there any test for this util? shall we create a few, just to be sure, since we are here?

Reply:
Yes. We should create a test first and then provide the change.